### PR TITLE
Cancel damage for dead players, resolves Electroid#179

### PR DESCRIPTION
**Pull request by Meeples10**
Signed-off-by: Meeples10 <8867705+Meeples10@users.noreply.github.com>

### CODE REVIEW

**Comment by Meeples10** [comment]
Is there a better place to cancel the event?

**Comment by TheMolkaPL** [comment]
This completly wrong place to do this. Trackers are meant to keep track of death reasons. Before you do the coding please try reproduce this bug before refacators. I belive one of them completly broke respawn system together with this and flying aroung bug.

**Comment by TheMolkaPL** [comment]
@Meeples10 can you check your Discord, please?

File: src/main/java/tc/oc/pgm/damage/DamageMatchModule.java [hunk 1]
```diff
@@ -159,6 +159,11 @@ public void onDamage(EntityDamageEvent event) {
     MatchPlayer victim = getVictim(event.getEntity());
     if (victim == null) return;
 
+    if (victim.isDead()) {
+      event.setCancelled(true);
+      return;
+    }
+
```
Comment (Pablete1234) [inline src/main/java/tc/oc/pgm/damage/DamageMatchModule.java]: I think this doesn't belong here, right?

**Pull request merged**